src/groundStation/groundStation.py

```diff
@@ -317,7 +317,7 @@ def getOptions(self):
             '-t',
             '--timeout',
             type=int,
-            default='10000', # 10 seconds
+            default='15000', # 15 seconds
```

Comment:
Anticipating this question. This was done because the burnwire command takes just a bit longer than 10seconds to respond, since the burnwires are physically burning for ~8seconds and we need to record the current to send down with this command

Reply:
ok